ports/raspberrypi/boards/pimoroni_interstate75/mpconfigboard.h

```diff
@@ -0,0 +1,41 @@
+#define MICROPY_HW_BOARD_NAME "Pimoroni Interstate 75"
+#define MICROPY_HW_MCU_NAME "rp2040"
+
+#define MICROPY_HW_R0 (&pin_GPIO0)
+#define MICROPY_HW_G0 (&pin_GPIO1)
+#define MICROPY_HW_B0 (&pin_GPIO2)
+#define MICROPY_HW_R1 (&pin_GPIO3)
+#define MICROPY_HW_G1 (&pin_GPIO4)
+#define MICROPY_HW_B1 (&pin_GPIO5)
+
+#define MICROPY_HW_ROW_A (&pin_GPIO6)
+#define MICROPY_HW_ROW_B (&pin_GPIO7)
+#define MICROPY_HW_ROW_C (&pin_GPIO8)
+#define MICROPY_HW_ROW_D (&pin_GPIO9)
+#define MICROPY_HW_ROW_E (&pin_GPIO10)
+
+#define MICROPY_HW_CLK (&pin_GPIO11)
+#define MICROPY_HW_LAT (&pin_GPIO12)
+#define MICROPY_HW_OE (&pin_GPIO13)
```

Comment:
What are these for? Is it an RGB matrix? I *think* you could init it in `board_init` to have it display the serial by default.

Reply:
Yes. HUB75 style panels. Tested already with the rgbmatrix library.

"Have it display the serial by default." Guessing you mean the matrix serial, not UART? Perhaps, but would that confuse things for anyone following guides?

Reply:
We haven't yet enabled an RGBMatrix as the built in display of a board, so there may be problems here.

Typically these panels are 64x32 at the greatest, leaving room for about 2 full rows of 6 characters or something; in my experience this hasn't been useful for viewing REPL errors.  And of course if it's just a header, not integrated with a specific size of matrix, the display might be jumbled depending what the board is plugged in to.

Reply:
The board is just a header connection (like MatrixPortal) so the problems you describe would likely be the case here too.